tests/data/snake_oil_data.py

```diff
@@ -108,6 +109,7 @@ def content(self):
     "http://127.0.0.1:5000/ensembles/2/parameters": [
         {"name": "test_parameter_1", "labels": []},
         {"name": "test_parameter_11", "labels": []},
+        {"name": "BPR_138_PERSISTENCE", "labels": []},
```

Comment:
Had to add an extra parameter to the second ensemble because I needed two ensembles with the same parameter